src/lib/engine/space.js

```diff
@@ -33,15 +34,21 @@ const user = (space, graph) => {
   return graph.users.find(e => sameIds(e.id, space.user_id))
 }
 
+const organization = (space, graph) => {
+  return graph.organizations.find(e => sameIds(e.id, space.organization_id))
+}
+
 export function toDgraph(spaceId, graph){
   let dGraph = _graph.denormalize(subset(graph, spaceId))
   const space = get(graph.spaces, spaceId)
   const spaceUser = user(space, graph)
+  const userOrganizationMemberships = graph.userOrganizationMemberships
+  const meId = _.get(graph, 'me.id')
   dGraph.user = spaceUser
-  dGraph.ownedByMe = sameIds(_.get(spaceUser, 'id'), _.get(graph, 'me.id'))
+  dGraph.ownedByMe = sameIds(space.user_id, meId) || UserOrganizationMemberships.isMember(space.organization_id, meId, userOrganizationMemberships)
```

Comment:
I would split this into a separate function, with userOrganizationMemberships, which only seems useful for this line.